Makefile

```diff
@@ -120,6 +120,8 @@ install-tools:
 # Install linters
 install-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b ${GO_PATH}/bin ${GOLANGCI_VERSION}
+	
+install-shfmt:
```

Comment:
It's strange, I've added it here:
 https://github.com/test-network-function/cnf-certification-test/commit/7906f8c9eee95fe03531d86a8c126fe371e5ad69

Reply:
Yeah I wanted to split it out because I use `brew` for my golangci-lint installations.

Reply:
There is a `brew` formula for `shfmt` as well.